buildSrc/src/main/java/Dependencies.kt

```diff
@@ -72,6 +73,7 @@ object FenixDependencies {
     const val kotlin_reflect = "org.jetbrains.kotlin:kotlin-reflect:${FenixVersions.kotlin}"
     const val kotlin_coroutines = "org.jetbrains.kotlinx:kotlinx-coroutines-core:${FenixVersions.coroutines}"
     const val kotlin_coroutines_test = "org.jetbrains.kotlinx:kotlinx-coroutines-test:${FenixVersions.coroutines}"
+    const val cashapp_turbine = "app.cash.turbine:turbine:${FenixVersions.cashapp_turbine}"
```

Comment:
I think this is gonna need a bit more scrutiny because adding a new dependency isn't a simple matter.

Reply:
Gotcha! Thanks for flagging, i will ask about this 👍